udev/CMakeLists.txt

```
@@ -1,3 +1,7 @@
 install(FILES 50-revpi.rules
   DESTINATION /lib/udev/rules.d
```

```suggestion
  DESTINATION ${CMAKE_INSTALL_LIBDIR}/udev/rules.d
```

Could you add a patch which fixes this as well, then we don't need to create an extra PR for this.

---

Sure thing

---

This will result in `/usr/lib/aarch64-linux-gnu/udev/rules.d` is that right? @hjokunbus @iluminat23